lib/constants.go

```diff
@@ -1560,6 +1560,11 @@ const (
 	DiamondLevelKey    = "DiamondLevel"
 	DiamondPostHashKey = "DiamondPostHash"
 
+	// Atomic Transaction Keys
+	AtomicTxnsChainLength    = "AtomicTxnsChainLength"
+	NextAtomicTxnPreHash     = "NextAtomicTxnPreHash"
+	PreviousAtomicTxnPreHash = "PreviousAtomicTxnPreHash"
+
```

Comment:
If we can shorten these constants while still having them be meaningful, that would be best. We enlarge the size each transaction w/ each of these, so the fewer bytes the better.